source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__calculation_results__energy_result_per_neighbor_ga_v1.sql

```diff
@@ -0,0 +1,14 @@
+CREATE VIEW {CALCULATION_RESULTS_DATABASE_NAME}.exchange_per_neighbor_ga_v1 as
+SELECT calculation_id,
+       calculation_type,
+       grid_area_code,
+       calculation_period_start,
+       calculation_period_end,
+       calculation_version,
```

Comment:
nit: I think it makes sense to have all the calculation columns first
```suggestion
       calculation_period_start,
       calculation_period_end,
       calculation_version,
       grid_area_code,
```
The comment applies to all

Reply:
okay, i will update confluence as well